File: python/chapter5/hello_admin.py
names = ["john", "jane", "admin", "jack", "jenny"]
names = list(str())

if len(names) != 0:
    for name in names:
        if name == "admin":
            print("Hello admin, would you like to see a status report?")
        else:
            print(f"Hello {name}, thank you for logging in again.")
else:
    print("We need to find some users!!")

File: javascript/chapter5/hello_admin.js
let names = ["john", "jane", "admin", "jack", "jenny"];
names = [];

if (names.length !== 0) {
	for (const name of names) {
		if (name === "admin") {
			console.log("Hello admin, would you like to see a status report?");
		} else {
			console.log(`Hello ${name}, thank you for logging in again.`);
		}
	}
} else {
	console.log("We need to find some users!!");
}

